openmrs-19/src/main/java/org/motechproject/openmrs19/resource/impl/PatientResourceImpl.java

```diff
@@ -132,44 +115,49 @@ public String getPatientIdentifierTypeUuidByName(String identifierTypeName) thro
     }
 
     @Override
-    public void deletePatient(String uuid) throws HttpException {
-        restfulClient.delete(openmrsInstance.toInstancePathWithParams("/patient/{uuid}?purge", uuid));
+    public void deletePatient(Config config, String uuid) {
+        delete(config, "/patient/{uuid}?purge", uuid);
     }
 
     @Override
-    public Patient updatePatientMotechId(String patientUuid, String newMotechId) throws HttpException {
-        Gson gson = new GsonBuilder().create();
-
-        Identifier patientIdentifier = getPatientIdentifier(patientUuid);
+    public void updatePatientMotechId(Config config, String patientUuid, String newMotechId) {
```

Comment:
I am still confused, how it should work. I see that we fetch the first identifier from list, previously we supported only one identifier (motechId), however now we can store also others. 

I think you can leave this, because there is a ticket to create update patient action, so then we will look deeper into this